#include "BraveNewWorldConfig.pb.h"
#include "AllianceConsumeConfig.pb.h"
#include "CompoundShredConfig.pb.h"
#include "RewardConfig.pb.h"
#include "GroupBuyingConfig.pb.h"
#include "ConfigScrollActivity.pb.h"
#include "ConfigKingTreasure.pb.h"
#include "ConfigScrollFeedback.pb.h"
#include "ConfigTokenUpgrade.pb.h"
#include "ConfigActivity.pb.h"
#include "TwoHolidayConfig.pb.h"
#include "ConfigFunction.pb.h"
#include "NewSoulStone.pb.h"
#include "YingLing.pb.h"
